packages/manager/apps/pci-load-balancer/src/pages/create/steps/region/RegionStep.spec.tsx

```diff
@@ -3,20 +3,19 @@ import { StepComponent, TStepProps } from '@ovh-ux/manager-react-components';
 import { render, renderHook } from '@testing-library/react';
 import { act } from 'react-dom/test-utils';
 import { OsdsLink, OsdsSpinner } from '@ovhcloud/ods-components/react';
-import {
-  ShapesInputComponent,
-  TShapesInputProps,
-} from '@ovh-ux/manager-pci-common';
 import { wrapper } from '@/wrapperRenders';
 import { RegionStep, TRegionStepProps } from './RegionStep';
-import { StepsEnum, TAddon, useCreateStore } from '@/pages/create/store';
+import { StepsEnum, useCreateStore } from '@/pages/create/store';
 import { REGION_AVAILABILITY_LINK } from '@/constants';
-import { TRegion } from '@/api/hook/useRegions';
-import { useTracking } from '../../hooks/useTracking';
-import { LabelComponent } from '@/pages/create/steps/region/components/Label.component';
-import { StackLabelComponent } from '@/pages/create/steps/region/components/StackLabel.component';
-import { StackTitleComponent } from '@/pages/create/steps/region/components/StackTitle.component';
-import { GroupLabelComponent } from '@/pages/create/steps/region/components/GroupLabel.component';
+
+vi.mock('@ovh-ux/manager-pci-common', () => ({
+  useProject: vi.fn().mockReturnValue({ data: { project_id: 'project_id' } }),
+  RegionSelector: () => <div />,
+  usePCICommonContextFactory: vi.fn(),
+  PCICommonContext: {
+    Provider: () => <></>,
+  },
+}));
```

Comment:
I think you should just mock the whole module and then import each element that you need to mock specifically